alpine-common/src/main/java/alpine/Config.java

```diff
@@ -181,7 +181,8 @@ public enum AlpineKey implements Key {
         CORS_ALLOW_CREDENTIALS                 ("alpine.cors.allow.credentials",     true),
         CORS_MAX_AGE                           ("alpine.cors.max.age",               3600),
         WATCHDOG_LOGGING_INTERVAL              ("alpine.watchdog.logging.interval",  0),
-        API_KEY_PREFIX                         ("alpine.api.key.prefix",             "alpine_");
+        API_KEY_PREFIX                         ("alpine.api.key.prefix",             "alpine_"),
+        AUTH_JWT_TTL                           ("alpine.auth.jwt.ttl",               7 * 24 * 60);
```

Comment:
Consider naming it `alpine.auth.jwt.ttl.seconds` to make it less ambiguous unit-wise.

Reply:
Should I rename `AUTH_JWT_TTL` too ?

Reply:
Yes please.